.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -57,7 +57,13 @@ body:
       description: |
         It can help us knowing the details of your ROS environment.
         Please use the command `ros2 doctor --report` and copy paste its output here.
-      render: Formatted
+      value: |

```

Comment:
this section is not required area, but i realize some issue reports are really long because of this. with this PR, we can expand the `ros2 doctor --report` section if needed, and keep the issue report simple.

Reply:
definitely!